### PR TITLE
Improve logos page

### PR DESCRIPTION
**Pull request by rikaznismi**

### CODE REVIEW

**Comment by rikaznismi** [comment]
Adds background to improve visibility for logos. Makes logos smaller for base and startup in order to include 6 logos in a row. Display what the company offers, e.g internship.

File: src/pages/logos.tsx [hunk 1]
```diff
@@ -4,26 +4,77 @@ import { addImageDetails } from "@/shared/addImageDetails";
...
 import { Package } from "@prisma/client";
 
+function OffersList({ offers }: { offers: (number[] | boolean)[] }) {
+  const jobTypeNames = [
+    "Summer Job",
+    "Internship",
+    "Part-time Job",
+    "Master Thesis",
+    "Full-time Job",
+    "Trainee Program",
```
Comment (ernum) [inline src/pages/logos.tsx:14]: Use locale for these (translation should already be available) to make these Swedish and English respectively.

File: src/pages/logos.tsx [hunk 2]
```diff
@@ -4,26 +4,77 @@ import { addImageDetails } from "@/shared/addImageDetails";
...
+    if (index >= 3 && offer) {
+      return true;
+    }
+    return false;
+  });
+
+  if (!hasValidOffer) return null;
+  return (
+    <div className="flex flex-col items-center mt-5">
+      <p className="text-cerise uppercase">Erbjuder</p>
```
Comment (ernum) [inline src/pages/logos.tsx:30]: This should also be translated

File: src/pages/logos.tsx [hunk 3]
```diff
@@ -129,34 +184,46 @@ export default function Logos({ exhibitorData }: LogosProps) {
   const startupPackages = exhibitorData.filter(
     (e) => e.package === "startup" && e.logoColor
   );
-
+  const baseAndStartUpPackages = [...basePackages, ...startupPackages];
+  const sponsorAndHeadhunterAndPremiumPackages = [
+    ...headhunterAndPremiumPackages,
+    ...sponsorPackages,
```
Comment (ernum) [inline src/pages/logos.tsx:190]: Do we want to have these together? Quite sure that headhunter and premium are more expensive than sponsor so they should probably be treated differently on this page

**Review by ernum — CHANGES_REQUESTED** [review]
Looks good, just a few small things

File: src/pages/logos.tsx (hunk 1 above)
Comment (rikaznismi) [reply src/pages/logos.tsx]: Fixed now

**Review by ernum — APPROVED** [review]
LGTM

**Pull request merged**